steps/docker.py

```diff
@@ -87,6 +106,16 @@ def step_impl(context, name):
     assert context.remote_cmd('shell',
                               module_args='docker ps -l | grep %s' % name)
 
+@then('check if "{matches}" is in result of docker run')
+def step_impl(context, matches):
+    '''check if string in the result of docker run'''
+    result = context.docker_run_result[0]
```

Comment:
The testing makes sure context.docker_run_result exists, but it makes sense to enhance check context.docker_run_result before using.